brayns/Brayns.cpp

```diff
@@ -324,81 +324,55 @@ struct Brayns::Impl : public PluginAPI
         const GeometryParameters& geometryParameters =
```

Comment:
so next commit when this code will be moved to the CircuitViewerPlugin? :smile:

Reply:
I already did it, but it was conflicting with Jonas' PR so I abandoned it. If you are interested in what I did, look at commits hernando/Brayns@eef533db and hernando/Brayns@e911400

Reply:
The plugin options we should discuss. They way I see it is that the PropertyMap can do all of that, also initializing them from commandline. The plugin can then define its own commandline options, and the core shouldn't know about them. Let's discuss once you're back.